dashboard/test/ui/features/sharepageLogo.feature

```diff
@@ -0,0 +1,47 @@
+@no_mobile
+@as_student
+Feature: Lab share page logo
+
+  Scenario: Select the logo on an applab share page while logged in and visit the homepage
+    Given I am on "http://studio.code.org/projects/applab"
+    And I wait for the page to fully load
+    Then I click selector ".project_share"
+    And I wait until element "#sharing-input" is visible
+    And I navigate to the share URL
+    And I wait for the page to fully load
+    And element "#logo-link" is visible
+    And I press "logo-link" to load a new page
+    And I wait until current URL contains "/studio.code.org/"
```

Comment:
Looks good. 

I don't think we necessarily need this since the word "STUDIO" is clearly removed in this PR. I'd be more concerned if it was still conditional, visible in some cases and not in others. Anyway, you could do something like:

```cucumber
And element "div:contains('STUDIO')" does not exist
```

See https://github.com/code-dot-org/code-dot-org/blob/f891026613d626fdfd6e80d5bb7daa24d773b138/dashboard/test/ui/step_definitions/steps.rb#L649-L651